src/main/asciidoc/inc/build/_configuration.adoc

```diff
@@ -8,6 +8,9 @@ of an image configuration. The following configuration options are supported:
 |===
 | Element | Description
 
+| <<config-image-build-assembly, *assemblies*>>
+| Specifies multiple assembly configurations as described in <<build-assembly,Build Assembly>>
+
```

Comment:
I think the enhanced assembly documentation is missing for the property style configuration. Could you please check and add it if missing there ?

Reply:
Are you talking about configuring assembly via properties? I'm trying this but somehow this isn't working

```
mvn docker:build -Ddocker.imagePropertyConfiguration=override \
  -Ddocker.from=fabric8/java-centos-openjdk8-jdk:1.5.6 \
  -Ddocker.cmd="java -jar /app/dmp-sample-multi-assembly-0.34-SNAPSHOT.jar" \
  -Ddocker.ports.1=8080 \
  -Ddocker.assemblies.1.descriptorRef=dependencies \
  -Ddocker.assemblies.1.name=deps \
  -Ddocker.assemblies.1.targetDir=/app/lib \
  -Ddocker.assemblies.2.descriptorRef=artifact \
  -Ddocker.assemblies.2.targetDir=/app
```